config/integration.yaml

```diff
@@ -1,6 +1,6 @@
 base:
   links:
-    - { 'v2.0': 'master' }
+    - { 'v2.1': 'master' }
```

Comment:
This will change the version dropdown to 2.1, and we will no longer have a 2.0.  Is this intended, should we branch out 2.1 from 2.0 if only for the minor changes in this PR?